### ISSUE TITLE
Loss graph

### ISSUE THREAD

**Issue by Henrik0808** [Henrik0808/ikt590-project#6]
While training a model, a loss graph should be generated that shows the loss for each epoch, both for the training dataset and the validation dataset. See example loss graph below.
The loss graph should be saved as a .png file in outputs/graphs/supervised_training/ for example, or for example in outputs/graphs/semi_supervised_training/phase_one/ or
outputs/graphs/semi_supervised_training/phase_two/

Example loss graph:
![example_loss_graph](https://user-images.githubusercontent.com/56306316/93755404-841fcf00-fc03-11ea-8f5f-767d0db689fd.png)

The loss graph can be used to detect overfitting, for example.
